sklearn/svm/_libsvm_sparse.pyx

```diff
@@ -43,50 +43,50 @@ cdef extern from "libsvm_sparse_helper.c":
     void copy_sv_coef   (char *, svm_csr_model *)
     void copy_n_iter  (char *, svm_csr_model *)
     void copy_support   (char *, svm_csr_model *)
-    void copy_intercept (char *, svm_csr_model *, np.npy_intp *)
-    int copy_predict (char *, svm_csr_model *, np.npy_intp *, char *, BlasFunctions *)
-    int csr_copy_predict_values (np.npy_intp *data_size, char *data, np.npy_intp *index_size,
-        	char *index, np.npy_intp *intptr_size, char *size,
+    void copy_intercept (char *, svm_csr_model *, cnp.npy_intp *)
+    int copy_predict (char *, svm_csr_model *, cnp.npy_intp *, char *, BlasFunctions *)
+    int csr_copy_predict_values (cnp.npy_intp *data_size, char *data, cnp.npy_intp *index_size,
+        	char *index, cnp.npy_intp *intptr_size, char *size,
                 svm_csr_model *model, char *dec_values, int nr_class, BlasFunctions *)
-    int csr_copy_predict (np.npy_intp *data_size, char *data, np.npy_intp *index_size,
-        	char *index, np.npy_intp *intptr_size, char *size,
+    int csr_copy_predict (cnp.npy_intp *data_size, char *data, cnp.npy_intp *index_size,
+        	char *index, cnp.npy_intp *intptr_size, char *size,
                 svm_csr_model *model, char *dec_values, BlasFunctions *) nogil
-    int csr_copy_predict_proba (np.npy_intp *data_size, char *data, np.npy_intp *index_size,
-        	char *index, np.npy_intp *intptr_size, char *size,
+    int csr_copy_predict_proba (cnp.npy_intp *data_size, char *data, cnp.npy_intp *index_size,
+        	char *index, cnp.npy_intp *intptr_size, char *size,
                 svm_csr_model *model, char *dec_values, BlasFunctions *) nogil
 
-    int  copy_predict_values(char *, svm_csr_model *, np.npy_intp *, char *, int, BlasFunctions *)
-    int  csr_copy_SV (char *values, np.npy_intp *n_indices,
-        	char *indices, np.npy_intp *n_indptr, char *indptr,
+    int  copy_predict_values(char *, svm_csr_model *, cnp.npy_intp *, char *, int, BlasFunctions *)
+    int  csr_copy_SV (char *values, cnp.npy_intp *n_indices,
+        	char *indices, cnp.npy_intp *n_indptr, char *indptr,
                 svm_csr_model *model, int n_features)
-    np.npy_intp get_nonzero_SV ( svm_csr_model *)
+    cnp.npy_intp get_nonzero_SV ( svm_csr_model *)
     void copy_nSV     (char *, svm_csr_model *)
-    void copy_probA   (char *, svm_csr_model *, np.npy_intp *)
-    void copy_probB   (char *, svm_csr_model *, np.npy_intp *)
-    np.npy_intp  get_l  (svm_csr_model *)
-    np.npy_intp  get_nr (svm_csr_model *)
+    void copy_probA   (char *, svm_csr_model *, cnp.npy_intp *)
+    void copy_probB   (char *, svm_csr_model *, cnp.npy_intp *)
+    cnp.npy_intp  get_l  (svm_csr_model *)
+    cnp.npy_intp  get_nr (svm_csr_model *)
     int  free_problem   (svm_csr_problem *)
     int  free_model     (svm_csr_model *)
     int  free_param     (svm_parameter *)
     int free_model_SV(svm_csr_model *model)
     void set_verbosity(int)
 
 
-np.import_array()
+cnp.import_array()
```

Comment:
I think we can remove this, because it is already called at the beginning of this file.

Reply:
Wooops, I missed this. Fixed.